webapp/src/dogma/features/project/settings/mirrors/MirrorForm.tsx

```diff
@@ -300,7 +300,7 @@ const MirrorForm = ({ projectName, defaultValue, onSubmit, isWaitingResponse }:
                 type="text"
                 defaultValue={defaultValue.remoteUrl}
                 placeholder="my.git.com/org/myrepo.git"
-                {...register('remoteUrl', { required: true, pattern: /^[\w.-]+(:[0-9]+)?\/[\w.-\/]+.git$/ })}
+                {...register('remoteUrl', { required: true, pattern: /^[\w.-]+(:[0-9]+)?\/[\w.\-\/]+.git$/ })}
```

Comment:
😅

Reply:
By the way, do we have to escape the first `-`?

Reply:
We don't have to because `-` is in the last position in `[]`. However, let me add it for clarification.